Makefile

```
@@ -18,11 +18,17 @@ format:
 format-check:
 	$(CRYSTAL) tool format --check
 
-lint:
-	@if [ ! -x bin/ameba ]; then \
-		echo "ameba not found at bin/ameba; run 'shards install' first" >&2; \
+# ameba no longer ships a postinstall that produces bin/ameba, so build it here
+# from the pinned checkout. The binary is rebuilt only when the source changes.
+bin/ameba: lib/ameba/src/cli.cr
```

checkmake (0.3.2)

[warning] Target body for "bin/ameba" exceeds allowed length of 5 lines (6).

(maxbodylength)